### PR TITLE
feat: warn when clang-format fails

### PR DESCRIPTION
**Pull request by kjin**

### CODE REVIEW

**Comment by ofrobots** [comment]
Odd that travis wasn't launched for this.

**Comment by kjin** [comment]
@ofrobots Probably because it's unmergeable... rebased

**Comment by DominicKramer** [comment]
LGTM: provided the tests pass

**Comment by ofrobots** [comment]
When landing, make sure the commit includes the 'Fixes: #60' metadata.

**Comment by kjin** [comment]
Since the approvals were issued, I added a test... was able to get code coverage to increase by [0.01%](https://codecov.io/gh/google/ts-style/pull/62)

A point of possible contention is [here](https://github.com/google/ts-style/pull/62/commits/f732709963e93f92434b569e2048ccb18a419d6c#diff-887ef50994f94df2ed539a9bdcb6a8e8R25). The explanation is that `exec` shouldn't actually be promisified the way it is because the callback may populate `err` _and_ `stdout`/`stderr`. So I added a new function to always resolve with an exit code (though most places still use the now-renamed "simple" `execp`)

Also, two things happened:
* `clang-format` updated, meaning things had to be re-formatted
* `chalk` got built-in TS definitions that clash with `@types/chalk`, so I removed the dependency on the latter (2e4711a)

@DominicKramer @ofrobots PTAL at the two linked things.

File: test/test-kitchen.ts [hunk 1]
```diff
@@ -8,14 +8,31 @@ import * as pify from 'pify';
...
 const pkg = require('../../package.json');
 
 const rimrafp = pify(rimraf);
 const mkdirp = pify(fs.mkdir);
-const execp = pify(cp.exec);
+const simpleExecp = pify(cp.exec);
 const renamep = pify(fs.rename);
 const ncpp = pify(ncp.ncp);
 
+const execp =
```
Comment (ofrobots) [inline test/test-kitchen.ts]: This deserves a comment. Also, we should probably look for an ecosystem module for execp.

**Comment by ofrobots** [comment]
Rule of thumb: if you have explain why some code is the way it is in the PR, you probably also need it as a comment.

LGTM once that is addressed.

**Pull request merged**